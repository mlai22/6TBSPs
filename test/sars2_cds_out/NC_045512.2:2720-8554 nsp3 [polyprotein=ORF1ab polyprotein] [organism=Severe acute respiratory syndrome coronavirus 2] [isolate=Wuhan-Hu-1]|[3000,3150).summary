NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	279	2.446672124323955e-34
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	279	2.446672124323955e-34
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	279	2.446672124323955e-34
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	279	2.446672124323955e-34
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	20	23.358337306992212
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	20	23.358337306992212
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725318.1:1-43 ORF7b [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	20	29.197921633740265
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	23	44.620477522664636
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	23	44.620477522664636
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	20	64.2354275942286
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	19	88.52525603066121
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	19	88.52525603066121
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	19	88.52525603066121
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	19	88.52525603066121
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	17	168.13275743662015
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	231.71006960892632
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	231.71006960892632
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	800.1425433106526
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	800.1425433106526
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	800.1425433106526
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	800.1425433106526
NC_045512.2:2720-8554 nsp3 [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]|[3000,3150)	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	800.1425433106526
