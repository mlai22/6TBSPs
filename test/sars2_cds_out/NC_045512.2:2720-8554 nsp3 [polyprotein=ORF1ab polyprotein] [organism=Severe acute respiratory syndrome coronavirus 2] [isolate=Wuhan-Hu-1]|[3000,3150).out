Frame: 1
Query: GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY
Length: 50
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 279
E-value: 2.446672124323955e-34

0     GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  49  
      ||||||||||||||||||||||||||||||||||||||||||||||||||      
1818  GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  1867


Frame: 1
Query: GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY
Length: 50
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 279
E-value: 2.446672124323955e-34

0     GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  49  
      ||||||||||||||||||||||||||||||||||||||||||||||||||      
1818  GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  1867


Frame: 1
Query: GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY
Length: 50
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 279
E-value: 2.446672124323955e-34

0     GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  49  
      ||||||||||||||||||||||||||||||||||||||||||||||||||      
1000  GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  1049


Frame: 1
Query: GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY
Length: 50
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 279
E-value: 2.446672124323955e-34

0     GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  49  
      ||||||||||||||||||||||||||||||||||||||||||||||||||      
1000  GTFTCASEYTGNYQCGHYKHITSKETLYCIDGALLTKSSEYKGPITDVFY  1049


Frame: -3
Query: RKHP
Length: 4
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 20
E-value: 23.358337306992212

1     KHP  3   
      |||      
5262  KHP  5264


Frame: -3
Query: RKHP
Length: 4
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 20
E-value: 23.358337306992212

1    KHP  3  
     |||     
870  KHP  872


Frame: 3
Query: YIYLC
Length: 5
Subject: YP_009725318.1:1-43 ORF7b [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 43

Score: 20
E-value: 29.197921633740265

2  YLC  4 
   |||    
9  YLC  11


Frame: 2
Query: VHLLVLVSTLVITSVVTINI
Length: 20
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 23
E-value: 44.620477522664636

3    LVLVSTLVITSVVTIN  18 
     |*|******||****|     
577  LRLIDAMMFTSDLATN  592

10    VITSVVTINI  19  
      |*|*|**||*      
1569  VFTTVDNINL  1578


Frame: 2
Query: VHLLVLVSTLVITSVVTINI
Length: 20
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 23
E-value: 44.620477522664636

3    LVLVSTLVITSVVTIN  18 
     |*|******||****|     
577  LRLIDAMMFTSDLATN  592

10    VITSVVTINI  19  
      |*|*|**||*      
1569  VFTTVDNINL  1578


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1273

Score: 20
E-value: 64.2354275942286

6    RTFVF  10 
     |*|||     
189  REFVF  193


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 19
E-value: 88.52525603066121

3    IRNRTF  8  
     **|*||     
807  VTNNTF  812

3    IRNRTFV  9  
     **|*||*     
807  VTNNTFT  813

0     VENIRNRTFV  9   
      |*||***|*|      
1573  VDNINLHTQV  1582

3    IRNRTFVF  10 
     **|*||**     
807  VTNNTFTL  814

0     VENIRNRTFVF  10  
      |***|****|*      
2426  VNGVRRSFYVY  2436


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 19
E-value: 88.52525603066121

3    IRNRTF  8  
     **|*||     
807  VTNNTF  812

3    IRNRTFV  9  
     **|*||*     
807  VTNNTFT  813

0     VENIRNRTFV  9   
      |*||***|*|      
1573  VDNINLHTQV  1582

3    IRNRTFVF  10 
     **|*||**     
807  VTNNTFTL  814

0     VENIRNRTFVF  10  
      |***|****|*      
2426  VNGVRRSFYVY  2436


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 19
E-value: 88.52525603066121

0     VENIRNRTFVF  10  
      |***|****|*      
1608  VNGVRRSFYVY  1618


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 19
E-value: 88.52525603066121

0     VENIRNRTFVF  10  
      |***|****|*      
1608  VNGVRRSFYVY  1618


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 17
E-value: 168.13275743662015

2    NIRNRTFV  9  
     *||***||     
464  DIRQLLFV  471


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 16
E-value: 231.71006960892632

8   FVF  10
    |||    
92  FVF  94


Frame: -1
Query: VENIRNRTFVF
Length: 11
Subject: YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 16
E-value: 231.71006960892632

8   FVF  10
    |||    
92  FVF  94


Frame: 2
Query: VHLLVLVSTLVITSVVTINI
Length: 20
Subject: YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1273

Score: 14
E-value: 800.1425433106526

17   INI  19 
     |||     
232  INI  234


Frame: 2
Query: VHLLVLVSTLVITSVVTINI
Length: 20
Subject: YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 14
E-value: 800.1425433106526

17   INI  19 
     |||     
280  INI  282


Frame: 2
Query: VHLLVLVSTLVITSVVTINI
Length: 20
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 14
E-value: 800.1425433106526

17    INI  19  
      |||      
1408  INI  1410


Frame: 2
Query: VHLLVLVSTLVITSVVTINI
Length: 20
Subject: YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 14
E-value: 800.1425433106526

17   INI  19 
     |||     
280  INI  282


Frame: 2
Query: VHLLVLVSTLVITSVVTINI
Length: 20
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 14
E-value: 800.1425433106526

17    INI  19  
      |||      
1408  INI  1410


